website/client/src/components/tasks/taskModal.vue

```diff
@@ -1244,6 +1269,23 @@ export default {
     selectedTags () {
       return this.getTagsFor(this.task);
     },
+    summarySentence () {
```

Comment:
this is not going to be translatable, right?

Reply:
For this version, correct. There's simply no way we could account for all possible sentence structures, with this format! If we decide we want to support other locales than `en`, we'll probably need to come up with a simpler `label: value` layout. But less than 10% of Group Plans (as determined by looking at the language preference of the group owner) use anything but `en`, so that's what we're running with for MVP.